docs-v2/features/otlp-export.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: 'OpenTelemetry export'
+---
+
+### Overview
+
+Nango provides OpenTelemetry export to help you monitor and analyze your integration activities. By exporting integration events as traces to your OpenTelemetry collector, you can gain valuable insights into your Nango usage and track various operations in real-time.
+
+<Info>
+**Prerequisites**
+- An active Nango Scale plan subscription
+- Access to an OpenTelemetry collector
+</Info>
+
+### Supported Events
+The following integration events are currently exported (as of November 2024):
+- Sync executions
+- Action executions
+- Webhook executions
+- Proxied requests
+
+### Configuration
+
+**Step 1**: Access your Environment settings
+
+Log into the Nango dashboard and navigate to the Environment settings page.
+
+**Step 2**: Configure OpenTelemetry Export
+
+In the Environment Settings, provide the following information:
+- OpenTelemetry endpoint: The endpoint URL of your OpenTelemetry collector. Ex: https://my.otlp.collector:4318/v1/
+- Headers (Optional): Any authentication headers or additional headers required to access your collector
```

Comment:
I would specify that headers are usually required for authorization purposes with a Datadog example.

Reply:
Datadog agent doesn't have a default header. It is up to whoever is running the agent to secure its access

Reply:
All good then!